src/qml/UnifiedPatchPage.qml

```diff
@@ -68,84 +83,89 @@ Page {
+                    _valueItem.wrapMode: Text.WrapAtWordBoundaryOrAnywhere
+                }
+            }
+            Separator {
+                    width: parent.width
+                    color: Theme.primaryColor
+                    horizontalAlignment: Qt.AlignHCenter
             }
-
             SectionHeader {
                 visible: modelData.conflicts.length > 0
-                text: qsTranslate("", "Possible conflicts")
+                text: qsTranslate("", "May conflict with:")
             }
-
-            Label {
-                color: Theme.highlightColor
-                anchors.left: parent.left
-                anchors.right: parent.right
-                anchors.margins: Theme.horizontalPageMargin
-                wrapMode: Text.WrapAtWordBoundaryOrAnywhere
-                text: modelData.conflicts.map(function(i) { return PatchManager.patchName(i) }).join("\n")
+            Repeater {
                 visible: modelData.conflicts.length > 0
-            }
-
-            SectionHeader {
-                text: qsTranslate("", "Version")
+                model: modelData.conflicts
+                delegate: TextSwitch {
+                    anchors.left: parent.left
+                    anchors.right: parent.right
+                    anchors.margins: Theme.horizontalPageMargin
+                    automaticCheck : false
+                    checked: PatchManager.isApplied(modelData)
+                    _label.color: checked ? Theme.primaryColor :  Theme.secondaryColor
+                    height: Math.max(Theme.itemSizeMedium, implicitHeight)
+                    text: PatchManager.patchName(modelData)
+                }
             }
 
             Label {
-                color: Theme.highlightColor
+                visible: PatchManager.developerMode && legacyPatch
+                color: Theme.primaryColor
                 anchors.left: parent.left
                 anchors.right: parent.right
                 anchors.margins: Theme.horizontalPageMargin
                 wrapMode: Text.WordWrap
-                text: modelData.rpm ? modelData.rpm : modelData.version
-            }
-
-            SectionHeader {
-                text: qsTranslate("", "Compatible")
-            }
-
-            Label {
-                color: Theme.highlightColor
-                anchors.left: parent.left
-                anchors.right: parent.right
-                anchors.margins: Theme.horizontalPageMargin
-                wrapMode: Text.WrapAtWordBoundaryOrAnywhere
-                text: modelData.compatible.join(', ')
+                horizontalAlignment: Text.AlignJustify
+                font.pixelSize: Theme.fontSizeSmall
+                text: qsTranslate("", "This patch uses the legacy format for its patch.json file. If you're the maintainer consider updating to the new format. If you are using Web Catalog you should not include a patch.json file at all in your upload.")
```

Comment:
> If you're the maintainer consider updating to the new format. If you are using Web Catalog you should not include a patch.json file at all in your upload

Is this possible? This is the set of files extensions allowed: https://github.com/CODeRUS/django-test/blob/master/example/demo/filehandler.py#L29

Reply:
As you have shown this is technically not possible.

Hence it is just the semantics of the human language (i.e., here: English)!
This is why I proposed [the use of RFC2119 language](https://github.com/sailfishos-patches/patchmanager/issues/78#issuecomment-934730025): Most "should" in Andrey's texts mean "must", though some of them really mean "should" and a few of them "may"!
The same with "shall", which sometimes is supposed to mean "must", but more often "should" (while it is definitely closer to "must" than "should" in English).

Reply:
The patch that 'triggered' this addition and the reworking of the Legacy page is https://coderus.openrepos.net/pm2/project/email-list-view-line-count which *does* include a legacy patch.json despite being a rather new patch.  
How it can be there?

> > If you're the maintainer consider updating to the new format. If you are using Web Catalog you should not include a patch.json file at all in your upload
> 
> Is this possible? This is the set of files extensions allowed: https://github.com/CODeRUS/django-test/blob/master/example/demo/filehandler.py#L29

It absolutely is, I just tested.

WC does actually allow a json file in the uploaded file, and does not replace/mangle it if it exists. How/why that check is bypassed I don't know.

This means you can actually ship a json file with information different than what was entered in WC. You then can e.g. have your patch listed as compatible in the Web Catalog list (because there data from the WC api call is used), but after installing you have the patch show up as red/incompatible in the patch list, because then information from the json file is used.

Reply:
Thanks for your review and changes. Much appreciated.  I shall try follow the style you propose in future changes.

> As you have shown, this is technically not possible. Hence it is simply subtly wrong use of semantics of a human language (i.e., here: English), hence the "should not" must be a "must not"!
> 
>[...]
> 
> Suggestion to resolve this here: Delete the whole third sentence, which cannot be the case, any way. 

But I have just shown, (if only to myself but anyone can easily verify), that it is actually possible to include any json file with the package you upload at WC. So the language describes the real-world situation even if it may be against the 
 "specification".

I would like to keep the sentence in because I believe that
 - we should encourage moving to the new format, because it has most importantly compatibility information which the legacy has not
 - the "nag" on the patch page is probably the place most visible to patch devs, who might not follow any changes on our docs site.

Reply:
Just wanted to add that the test I linked to, while testing `.endswith(extensions)` passes because `".json".endswith(".js")`:)